pkg/cmd/cli/cmd/logs_test.go

```diff
@@ -1,29 +0,0 @@
-package cmd
-
-import (
-	"io/ioutil"
-	"reflect"
-	"testing"
-
-	"github.com/spf13/pflag"
-
-	kcmd "k8s.io/kubernetes/pkg/kubectl/cmd"
-)
-
-// TestFlagParity makes sure that our copied flags don't slip during rebases
-func TestFlagParity(t *testing.T) {
-	kubeCmd := kcmd.NewCmdLog(nil, ioutil.Discard)
-	originCmd := NewCmdLogs("", nil, ioutil.Discard)
-
-	kubeCmd.LocalFlags().VisitAll(func(kubeFlag *pflag.Flag) {
```

Comment:
You still need this check to make sure that we don't have flag conflicts as we start adding our own log options.